### PR TITLE
[Reviewer RKD] Add a comment to alarms.py.

### PR DESCRIPTION
**Pull request by chris-elford-metaswitch**
Rob - I think this sums up the conversation we just had. Any comments before I merge?

### CODE REVIEW

File: metaswitch/common/alarms.py [hunk 1]
```diff
@@ -46,6 +46,10 @@
 sendrequest = None
 
 def issue_alarm(process, identifier):
+    # Clearwater nodes all have clearwater-infrastructure installed.
+    # It includes a command-line script that can be used to issue an alarm.
```
Comment (rkd-msw) [inline metaswitch/common/alarms.py]: I might add a link to https://github.com/Metaswitch/clearwater-infrastructure/blob/master/clearwater-infrastructure/usr/share/clearwater/bin/alarms.py

File: metaswitch/common/alarms.py [hunk 2]
```diff
@@ -46,6 +46,10 @@
...
+    # Rather than refactor clearwater-infrastructure, we re-use the function
```
Comment (rkd-msw) [inline metaswitch/common/alarms.py]: I'm wholly in favour of this new passive-aggressive approach to writing comments, and intend to incorporate it into my own style.

Comment (chris-elford-metaswitch) [inline metaswitch/common/alarms.py]: I look forward to it ;)

**Comment by chris-elford-metaswitch** [comment]
I've fixed this up and pushed.

**Pull request merged**